src/pocketmine/item/Item.php

```diff
@@ -821,6 +821,17 @@ public function setCount(int $count){
 		return $this->canBePlaced();
 	}
 
+	public function canBeConsumed() : bool{
```

Comment:
This function can be made `protected`, since only `canBeConsumedBy` should be called externally.